Owlcat/bin/flag-ms.py

```diff
@@ -531,6 +535,25 @@ def parse_subset_options(options):
                         nvis_A, nvis_A * percent))
                 print(
                     "===>   %-29s includes %10d visibilities (%.3g%% of selection)" % (label, nv, nv * percent))
+                    
+            # now check for stats over CASA flagversions
+            flagvers = [tab for tab in glob.glob(f"{msname}.flagversions/flags.*") if Owlcat.tableexists(tab)]
+            if flagvers:
+                print(f"Stats for CASA-style flagversions:")
+                for vers in flagvers:
+                    name = os.path.basename(vers).split('.', 1)[1]
+                    ftab = Owlcat.table(vers, ack=False)
+                    flag_row = ftab.getcol("FLAG_ROW")
```

Comment:
suggest chunked reads. on large 32k datasets this column can be quite massive

Reply:
All right you convinced me... See latest commit.